### ISSUE TITLE
Unlocalized system strings

### ISSUE THREAD

**Issue by Saegusae** [Foglio1024/Tera-custom-cooldowns#163]
There are some unlocalized (dc driven) system strings left in the chat sub window. For example leveling Weaponcrafting yields.

```
@artisan:21 increased by 5.0
```

**Comment by Foglio1024** [comment]
Yeah I know, I'm handling only some specific strings, to avoid parsing other stuff from DC (also because my message parser class should be refactored but it's huge and I'm too lazy).
